nautobot/core/views/renderers.py

```diff
@@ -163,18 +156,11 @@ def get_context(self, data, accepted_media_type, renderer_context):
             form = data["form"]
         else:
             if view.action == "list":
-                filter_params = self.get_filter_params(view, request)
                 if view.filterset_class is not None:
+                    view.queryset = view.filter_queryset(view.queryset)
+                    filter_params = view.get_filter_params(request)
                     filterset = view.filterset_class(filter_params, view.queryset)
```

Comment:
Seems a bit awkward that we instantiate `filterset` at least twice in this flow (once in `view.filter_queryset` and once here)... wonder if there's any cleaner way we could do this? Maybe setting a `view.filterset` attribute or something?

Similar nit about calling `get_filter_params` twice.